iidm/iidm-impl/src/main/java/com/powsybl/iidm/network/impl/AbstractConnectable.java

```diff
@@ -49,7 +50,7 @@ void addTerminal(TerminalExt terminal) {
 
     @Override
     public NetworkImpl getNetwork() {
-        return networkRef.get();
+        return Optional.ofNullable(networkRef).map(Ref::get).orElse(null);
```

Comment:
One detail, isn't it less clear to create an `Optional` instead of doing the classical ternary `return x!=null ? y(x) : null`, or  of writing `if x equals null return null`?